youtube_dl/extractor/dreisat.py

```diff
@@ -34,54 +28,6 @@ class DreiSatIE(InfoExtractor):
     def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         video_id = mobj.group('id')
-        details_url = 'http://www.3sat.de/mediathek/xmlservice/web/beitragsDetails?ak=web&id=%s' % video_id
-        details_doc = self._download_xml(details_url, video_id, 'Downloading video details')
-
-        status_code = details_doc.find('./status/statuscode')
```

Comment:
this part is not handled by ZDFIE, i will move it to extract_from_xml_url in zdf.py.